src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -1,11 +1,20 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToMany;
 import java.util.ArrayList;
 import java.util.List;
 
+@Entity
 public class Movie implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String title;
+    @ManyToMany
     private List<Actor> actors;
```

Comment:
It's good practice to initialize collections in entities to prevent `NullPointerException`. You can initialize the `actors` list in the field declaration (e.g., `private List<Actor> actors = new ArrayList<>();`).